### ISSUE TITLE
scroll to hide/show toolbar 

### ISSUE THREAD

**Issue by oillipheist** [kdwk/Spidey#53]
hide/show toolbar on scroll would increase usability on touch devices immensely. when testing on steamdeck i need to use trackpad and mouse pointer to trigger a reveal.

**Comment by kdwk** [comment]
Good idea. When I have time I'll investigate how to define gestures separately for touchpad and touch.
